### DISCUSSION TITLE
Deta Deployment Page is Outdated

### DISCUSSION THREAD

**Discussion by Suvoo** [fastapi/fastapi#9284] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
PS C:\Users\91865\Desktop\FasDEp> deta new
Error: failed to create new program: Operation no longer supported, see https://go.deta.dev/migrate/deta-new for more details.
```

### Description

The current documentation for the deployment of Fast API apps on Deta is [outdated](https://fastapi.tiangolo.com/deployment/deta/). They have switched to "Deta Space" and all previous commands have to be replaced with [newer versions](https://deta.space/docs/en/introduction/first-app).

Instead of `deta new`, the new version is `space new` and the same goes for push and release.

This can be quite confusing for beginners looking to deploy, as it was the case with me.
@tiangolo 

### Operating System

Windows

### FastAPI Version

0.94.1

### Python Version

3.9.1

**Comment by Kludex** [comment, marked as answer]
Deta page was removed.
